mscore/importmxmlpass1.cpp

```diff
@@ -542,7 +542,7 @@ static void doCredits(Score* score, const CreditWordsList& credits, const int pa
              pf->width(), pf->height(), pf->oddTopMargin(), score->spatium(), DPMM, DPI);
       */
       // page width, height and odd top margin in tenths
-      const double ph  = score->styleD(Sid::pageHeight) * 10 * DPI / score->spatium();
+      const double ph = score->styleI(Sid::pageFullHeight) * 10 / MSCX_F / score->spatium();
```

Comment:
again magic 10

Reply:
Yes, but it's in a different module :)  I'll have to make the new function public and make the class available here too.